### PR TITLE
adding a First Steps workflow

### PR DESCRIPTION
**Pull request by joergbuchwald**
First Steps documentation. Based on my personal experiences ;-)

### CODE REVIEW

**Comment by nagelt** [comment]
very helpful ...

File: web/content/docs/tools/getting-started/first_steps/index.pandoc [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    parent = "getting-started"
++++
+
+## Frist Steps
+
+This section describes a possible general pre-processing workflow using some of the tools provided by OGS.
+
+To set up a model domain, the project file `*.prj` requires additional input files for the mesh and geometry to apply the boundary conditions accordingly.
+
+The mesh must be provided in VTK's `*.vtu` format, whereas the geometry can be provided either in OGS' internal `*.gml` file format (not to be confused with the Geography Markup Language) or as a `*.vtu` file as well, containing boundary elements only (Multiple `*.vtu` files can be provided in the project file using the `<meshes>` tag). We recommend the usage of the first method for simple 2D meshes with constant boundary conditions, whereas more complicated geometries and conditions might require the latter method. One general advatage in the utilization of `*.vtu` files is that they allow for defining additional field variables at each mesh node/element in order to implement spatially varying boundary conditions in a similar manner as defining inhomogeneous meterial properties.
```
Comment (TomFischer) [inline web/content/docs/tools/getting-started/first_steps/index.pandoc]: spelling: meterial -> material

**Comment by TomFischer** [comment]
Nice! :+1:

File: web/content/docs/tools/getting-started/first_steps/index.pandoc [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+[menu]
+  [menu.tools]
+    parent = "getting-started"
++++
+
+## Frist Steps
+
+This section describes a possible general pre-processing workflow using some of the tools provided by OGS.
+
+To set up a model domain, the project file `*.prj` requires additional input files for the mesh and geometry to apply the boundary conditions accordingly.
```
Comment (endJunction) [inline web/content/docs/tools/getting-started/first_steps/index.pandoc]: comma missing:
```
mesh, and geometry to ...
```

File: web/content/docs/tools/getting-started/first_steps/index.pandoc (hunk 1 above)
Comment (endJunction) [inline web/content/docs/tools/getting-started/first_steps/index.pandoc]: spelling: advantage.
wording: "allow for defining additional" -> "allow a definition of additional"

**Comment by endJunction** [comment]
Links are not rendered correctly. Have a look into the `details` link for `deploy/netlify`.

**Comment by joergbuchwald** [comment]
> Links are not rendered correctly. Have a look into the `details` link for `deploy/netlify`.

What did I do wrong? It's the same syntax as in the other files.

File: web/content/docs/tools/getting-started/first_steps/index.pandoc (hunk 1 above)
Comment (joergbuchwald) [reply web/content/docs/tools/getting-started/first_steps/index.pandoc]: thx

Comment (joergbuchwald) [reply web/content/docs/tools/getting-started/first_steps/index.pandoc]: thx

File: web/content/docs/tools/getting-started/first_steps/index.pandoc (hunk 2 above)
Comment (joergbuchwald) [reply web/content/docs/tools/getting-started/first_steps/index.pandoc]: I don't think so. Maybe without 'the' in front of 'mesh'.

Comment (endJunction) [reply web/content/docs/tools/getting-started/first_steps/index.pandoc]: Sorry, then I didn't understand what you were saying. Just me, I guess. The sentence is maybe too long and tries to cover several things at once: the meshes for the project, and then other meshes/geometries for BCs.
Never mind the comma.

**Comment by bilke** [comment]
@joergbuchwald Thanks for your contribution! I fixed the links in this [PR](https://github.com/joergbuchwald/ogs/pull/1). Please merge it.

**Pull request merged**